apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -113,6 +111,9 @@ defmodule OMG.Watcher.BlockGetter do
          } = to_apply},
         state
       ) do
+    exit_processor_results = ExitProcessor.check_validity()
```

Comment:
Looks dangerous as it changes the flow of processing exits. Have you considered this carefully :question: